test/parallel/test-querystring.js

```diff
@@ -204,22 +203,22 @@ qsNoMungeTestCases.forEach(function(testCase) {
 // now test stringifying
 
 // basic
-qsTestCases.forEach(function(testCase) {
-  assert.strictEqual(testCase[1], qs.stringify(testCase[2]));
-});
+qsTestCases.forEach((testCase) =>
+  assert.strictEqual(testCase[1], qs.stringify(testCase[2]))
+);
 
-qsColonTestCases.forEach(function(testCase) {
-  assert.strictEqual(testCase[1], qs.stringify(testCase[2], ';', ':'));
-});
+qsColonTestCases.forEach((testCase) =>
+  assert.strictEqual(testCase[1], qs.stringify(testCase[2], ';', ':'))
+);
 
-qsWeirdObjects.forEach(function(testCase) {
-  assert.strictEqual(testCase[1], qs.stringify(testCase[0]));
-});
+qsWeirdObjects.forEach((testCase) =>
+  assert.strictEqual(testCase[1], qs.stringify(testCase[0]))
+);
 
 // invalid surrogate pair throws URIError
-assert.throws(function() {
+assert.throws(() => {
   qs.stringify({ foo: '\udc00' });
-}, URIError);
+}, common.expectsError('ERR_URI_MALFORMED', URIError, 'URI malformed'));
```

Comment:
The `expectsError` arguments seem to be faulty. Please compare them to other usages or look them up in the function definition.